src/libraries/Native/Unix/System.Security.Cryptography.Native/apibridge.c

```diff
@@ -48,6 +48,38 @@ const ASN1_TIME* local_X509_get0_notAfter(const X509* x509)
     return NULL;
 }
 
+int local_X509_set1_notBefore(X509* x509, const ASN1_TIME* time)
+{
+    if (x509 && x509->cert_info && x509->cert_info->validity)
+    {
+        if (x509->cert_info->validity->notBefore)
+        {
+            ASN1_TIME_free(x509->cert_info->validity->notBefore);
+        }
+
+        x509->cert_info->validity->notBefore = ASN1_STRING_dup(time);
```

Comment:
This should be wrapped in `if (x509->cert_info->validity->notBefore != time)`, both for side effect observability reasons, and to avoid `X509_set1_notBefore(cert, X509_get0_notBefore(cert))` from segfaulting.